src/style.css

```diff
@@ -1,3 +1,88 @@
+/* stylelint-disable-next-line import-notation */
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap');
+
 body {
+  box-sizing: border-box;
   margin: 0;
 }
+
+.header {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+  justify-content: space-between;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  height: 60px;
+  justify-content: space-between;
+  align-items: center;
+  list-style: none;
+}
+
+.header__logo {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav {
+  display: flex;
+  max-height: 60px;
+}
+
+.nav__link {
+  position: relative;
+  display: inline-block;
+  padding: 23px 0;
+  text-decoration: none;
+  height: 100%;
+  color: black;
+}
+
+.nav__link:first-of-type {
+  padding-left: 0;
+}
+
+.nav__item {
+  font-family: roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  text-transform: uppercase;
+  font-size: 12px;
+  margin: 0 10px;
+}
+
+.nav__item:last-of-type {
+  padding-right: 0;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
```

Comment:
you can remove this after adding reset.css

Reply:
ok

Reply:
missed this comment